djl-spring-boot-console-sample/pom.xml

```diff
@@ -9,7 +9,7 @@
     </parent>
     <groupId>ai.djl.spring.examples</groupId>
     <artifactId>djl-spring-boot-console-sample</artifactId>
-    <version>0.12-SNAPSHOT</version>
+    <version>0.14.0</version>
```

Comment:
```suggestion
    <version>0.14.0-SNAPSHOT</version>
```

Reply:
This should be `-SNAPSHOT`, removing -SNAPSHOT will cause failing to publish -SNAPSHOT version to maven. maven publish plugin will remove -SNAPSHOT automatically when publish to release repo